### PR TITLE
Init poweremail docs

### PR DESCRIPTION
**Pull request by jaumef**
### Description:

Add PowerEmail documentation explaining module structure and usability

### Tasks:

- [x] Add Poweremail Docs
- [x] Translate

### Links:

* CA [Base/PowerEmail](http://builds.gisce.net/powerp-docs/powerp_add_poweremail_docs/ca/base/poweremail/)
* ES [Base/PowerEmail](http://builds.gisce.net/powerp-docs/powerp_add_poweremail_docs/es/base/poweremail/)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_poweremail_docs/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_poweremail_docs/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_poweremail_docs/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_poweremail_docs/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_poweremail_docs/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/base/poweremail.md [hunk 1]
```diff
@@ -1 +1,144 @@
...
+[compte PowerEmail aprobat](#aprobar-un-compte-poweremail), podem suspendre el
+compte per a que quedi en estat `suspesa`.
+
+![](_static/poweremail/pwmail_account_suspend.png)
+
+### Reactivació d'un compte PowerEmail
+
+Quan un compte està suspés, és pot reactivar mitjançant el botó `solicitar
+reactivació` fent que passi al estat `esborrany` tornant a habilitar el botó
+`Abrobar el compte`.
```
Comment (MiquelIR) [inline docs/base/poweremail.md]: Typo in `Abrobar el compte`.

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_poweremail_docs/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_poweremail_docs/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_poweremail_docs/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_poweremail_docs/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_poweremail_docs/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_poweremail_docs/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/base/poweremail.md [hunk 2]
```diff
@@ -1 +1,316 @@
...
+reactivació` fent que passi al estat `esborrany` tornant a habilitar el botó
+`Aprobar el compte`.
+
+![](_static/poweremail/pwmail_account_reactivate.png)
+
+## Gestió de Bústies de correu
+
+Segons la configuració utilitzada en la [creació d'un compte PowerEmail](#creacio-dun-compte-poweremail),
+Els correus enviats o rebuts es trobaran en el menú de bústies "`Personal`", quan
+el compte està assignat al nostre usuari, o "`Companyia`", quan el nostre usuari
```
Comment (gdalmau) [inline docs/base/poweremail.md]: (es) `Compañía"` → `"Compañía`

File: docs/base/poweremail.md [hunk 3]
```diff
@@ -1 +1,316 @@
...
+També cal destacar el botó `Enviar Correu` que permet el reenviament del correu.
+
+En la pestanya _Adjunts_ es poden trobar els diferents fitxers adjunts que s'han
+enviat o rebut en el correu electrònic.
+
+En la pestanya _Raw Content_ es troba tot el correu electrònic en format de text pla.
+
+En la pestanya _Avançat_ hi trobem les configuracions relacionades amb el compte
+de PowerEmail i amb les tasques realitzades amb aquest correu en l'historial
+(vegades enviat, rebut o mogut de bústia).
```
Comment (gdalmau) [inline docs/base/poweremail.md]: (es) `buzon` → `buzón`

File: docs/base/poweremail.md [hunk 4]
```diff
@@ -1 +1,316 @@
...
+    Aquesta pestanya conté dades relacionades amb el tipus de missatge a enviar,
+    el compte a utilitzar i referéncies del servidor de correu.
+    Per aquest motiu, tot i que es puguin editar els camps, no és aconsellable
+    canviar-los.
+
+### Converses PowerEmail
+
+El mòdul poweremail permet mantenir converses de correu electrònic, és a dir,
+detecta quan un correu és una resposta a un dels correus enviats o si un correu
+enviat és la resposta d'un rebut.
```
Comment (gdalmau) [inline docs/base/poweremail.md]: (es) `recibido` → `recibido.` (falta el punt final)

File: docs/base/poweremail.md [hunk 5]
```diff
@@ -1 +1,316 @@
...
+
+Aquests correus es troben en una bústia especial en la carpeta "**Personal**".
+
+Símplement es mostra una llista de les converses que s'ha tingut, amb quin usuari
+i quants correus hi ha a la conversa.
```
Comment (gdalmau) [inline docs/base/poweremail.md]: (es) `conversacion` → `conversación.` (falta el punt final i l'accent)

File: docs/base/poweremail.md [hunk 6]
```diff
@@ -1 +1,316 @@
...
+
+Si entrem en una conversa únicament es mostra una llista de correus que són els
+que estan relacionats amb la conversa.
```
Comment (gdalmau) [inline docs/base/poweremail.md]: (es) `conversación` → `conversación.` (falta el punt final)

File: docs/base/poweremail.md [hunk 7]
```diff
@@ -1 +1,316 @@
...
+
++------------------------+------------------------------------------------------------+
+|      Nom del camp      | Descripció                                                 |
++========================+============================================================+
+|   Nom de la plantilla  | Referència a utilitzar per aquesta plantilla.              |
++------------------------+------------------------------------------------------------+
+|           Model        | Model relacionat amb aquesta plantilla.                    |
+|                        | Permet l'ús de camps d'aquest model.                       |
++------------------------+------------------------------------------------------------+
+| Idioma de la plantilla | Llenguatge utilitzat en la generació del correu des de les |
```
Comment (gdalmau) [inline docs/base/poweremail.md]: (es) `Lenguage` → `Lenguaje`

File: docs/base/poweremail.md [hunk 8]
```diff
@@ -1 +1,316 @@
...
+|                        | dades de la plantilla                                      |
++------------------------+------------------------------------------------------------+
+
+Cal emplenar aquests camps per utilitzar les plantilles. Juntament amb els camps
+referents al correu a enviar, com per a qui seria el correu, l'assumpte o el cos
```
Comment (gdalmau) [inline docs/base/poweremail.md]: (es) `seria` → `sería`

File: docs/base/poweremail.md [hunk 9]
```diff
@@ -1 +1,316 @@
...
+del missatge.
+
+!!! Tip "Consell"
+    Es pot utilitzar l'assistent d'expressions per generar el codi per obtenir
+    el valor d'un camp del model de la plantilla.
```
Comment (gdalmau) [inline docs/base/poweremail.md]: (es) `el valos` → `el valor`

File: docs/base/poweremail.md [hunk 10]
```diff
@@ -1 +1,316 @@
...
+
+    ![](_static/poweremail/pwmail_template_new_expression.png)
+
+Tots els camps referents a la generació del correu (adreces i cos del missatge)
+són analitzats per si són del llenguatge de la plantilla, és a dir, s'hi pot
```
Comment (gdalmau) [inline docs/base/poweremail.md]: `s'hi pot` → `s'hi poden`
(es) `se puede utilizar las expresiones` → `se pueden utilizar las expresiones`
(es) `són` → `son`

File: docs/base/poweremail.md [hunk 11]
```diff
@@ -1 +1,316 @@
...
+utilitzar les expressions generades en l'assistent.
+
+La pestanya de seguretat, permet definir quins usuaris poden utilitzar aquesta
+plantilla per enviar correus.
+
+Per altra banda, a la pestanya `Avançat` hi podem trobar diverses utilitats.
+
+![](_static/poweremail/pwmail_template_new_advanced.png)
+
+En la primera secció, podem configurar automatismes de cares a l'enviament del
```
Comment (gdalmau) [inline docs/base/poweremail.md]: (es) `envio` → `envío`

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_poweremail_docs/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_poweremail_docs/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by jaumef** [comment]
@gdalmau solved!
:cake:

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_poweremail_docs/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by gdalmau** [comment]
@jaumef Be careful when using replace while editing translations with poedit:

- Replacing `buzon` → `buzón` produces `buzones` → `buzónes`
- Replacing `segun` → `según` produces `Segunda` → `Segúnda`

Fix these strings please.

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_poweremail_docs/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**